apps/launch/components/MyStakingContracts/index.tsx

```diff
@@ -21,66 +22,96 @@ const ICON_STYLE = { fontSize: '56px', color: '#A3AEBB' };
 const StyledMain = styled.main`
   display: flex;
   flex-direction: column;
-  max-width: 1000px;
+  max-width: 1200px;
   margin: 0 auto;
 `;
 
+const Loader = () => (
+  <Flex justify="center" align="center" style={{ height: '40vh' }}>
+    <Spin />
+  </Flex>
+);
+
 const ConnectWallet = () => {
   return (
-    <Flex align="center" vertical gap={16}>
-      <WalletOutlined style={ICON_STYLE} />
-      <Paragraph type="secondary" className="text-center">
-        Connect your wallet to allocate your voting power.
-      </Paragraph>
-      <LoginV2 />
-    </Flex>
+    <>
+      <CreateContractMessage />
+      <Flex align="center" vertical gap={16}>
+        <WalletOutlined style={ICON_STYLE} />
+        <Paragraph type="secondary" className="text-center">
+          Connect your wallet to allocate your voting power.
+        </Paragraph>
+        <LoginV2 />
+      </Flex>
+    </>
   );
 };
 
-const StakingContractList = () => {
+const CreateContractMessage = () => (
+  <Paragraph type="secondary" className="mt-8 mb-16">
+    Create staking contracts to get agents running in your ecosystem.
+  </Paragraph>
+);
+
+const CreateContractButton = () => {
   const router = useRouter();
   const { networkName } = useAppSelector((state) => state.network);
-  console.log('networkName', `${networkName}/${URL.myStakingContracts}/create`);
 
   return (
+    <Button
```

Comment:
I think we can just use Link with Button inside, so no need to use router hook and do push, wdyt?
```
<Link href={`/${networkName}/${URL.myStakingContracts}/create`} passHref>
    <Button>Create staking contract</Button>
</Link>
```

Reply:
1) less code
2) no need to use route hook (doesn't change much though, but could in case we have e.g. filtering that changes the url)
3) on each component render the onClick function won't be recreated (cuz there will be no onClick passed)